.copier-answers.yml

```diff
@@ -1,6 +1,6 @@
 # Do NOT update manually; changes here will be overwritten by Copier
 _commit: v1.20
-_src_path: gh:oca/oca-addons-repo-template
+_src_path: git+https://github.com/oca/oca-addons-repo-template
```

Comment:
Don't change this.

Reply:
Not my personal change, `pipx run copier copy git+https://github.com/oca/oca-addons-repo-template . --trust` did that

Reply:
I mean, I'll revert yet what did I do wrong?

Reply:
You have to run it with `pipx run copier copy gh://OCA/oca-addons-repo-template . --trust`

Reply:
Done